presidio-analyzer/tests/test_us_driver_license_recognizer.py

```diff
@@ -40,6 +40,10 @@ def entities():
         ),
         ("ABCDEFG ABCDEFGH ABCDEFGHI", 0, (), (),),
         ("ABCD ABCDEFGHIJ", 0, (), (),),
+        ("Grant us the rights to use your contribution. For details, "
+         "visit https://cla.microsoft.com or contact (212) 555-1234.", 0, (), (),),
+        ("Grant us the rights to use your CONTRIBUTION. For details, "
+         "visit https://cla.microsoft.com or contact (212) 555-1234.", 1, ((32, 44),), ((0.3, 0.4),),),
```

Comment:
@omri374 Plain 'CONTRIBUTION' or 'contribution' isn't working for some reason, although working in https://regex101.com/r/NgGEbq/1
Something about the escaping / lookaround, lets review it in seperate